modules/lang-mustache/src/main/java/org/elasticsearch/rest/action/search/template/RestPutSearchTemplateAction.java

```diff
@@ -25,42 +25,41 @@
 import org.elasticsearch.rest.RestChannel;
 import org.elasticsearch.rest.RestController;
 import org.elasticsearch.rest.RestRequest;
+import org.elasticsearch.rest.action.admin.cluster.storedscripts.RestPutStoredScriptAction;
 import org.elasticsearch.script.Template;
 
 import static org.elasticsearch.rest.RestRequest.Method.POST;
 import static org.elasticsearch.rest.RestRequest.Method.PUT;
 
-/**
- *
- */
 public class RestPutSearchTemplateAction extends RestPutStoredScriptAction {
 
     @Inject
     public RestPutSearchTemplateAction(Settings settings, RestController controller, Client client) {
         super(settings, controller, false, client);
-
-        //controller.registerHandler(GET, "/template", this);
         controller.registerHandler(POST, "/_search/template/{id}", this);
         controller.registerHandler(PUT, "/_search/template/{id}", this);
+        controller.registerHandler(PUT, "/_search/template/{id}/_create", new CreateHandler(settings, client, this));
```

Comment:
Actually this can be removed, because we don't have the create semantic with stored scripts. Also the create handlers can be removed in `RestPutStoredScriptAction `. I forgot to remove this when we moved from indexed scripts to stored scripts.